examples/ollama_plugin/ollama_plugin/serve_llm.py

```diff
@@ -0,0 +1,98 @@
+# %% [markdown]
+# (serve_llm)=
+#
+# # Serve LLMs with Ollama
+#
+# In this guide, you'll learn how to locally serve Gemma2 and fine-tuned Llama3 models using Ollama within a Flyte task.
+#
+# Start by importing Ollama from the `flytekitplugins.inference` package and specifying the desired model name.
+#
+# Below is a straightforward example of serving a Gemma2 model:
+# %%
+from flytekit import ImageSpec, Resources, task
+from flytekit.extras.accelerators import A10G
+from flytekitplugins.inference import Model, Ollama
+from openai import OpenAI
+
+image = ImageSpec(
+    name="ollama_serve",
+    registry="ghcr.io/flyteorg",
+    packages=["flytekitplugins-inference"],
+    builder="default",
+)
+
+ollama_instance = Ollama(model=Model(name="gemma2"))
+
+
+@task(
+    container_image=image,
+    pod_template=ollama_instance.pod_template,
+    accelerator=A10G,
+    requests=Resources(gpu="0"),
```

Comment:
Does this work with `gpu="0"`?

Reply:
yes, it works with gpu="0". the gpu is allocated to the ollama server.

Reply:
It's not obvious from reading the code that `Ollama` has a default of one GPU. Can we add `gpu=1` in the Ollama definition.

Also, does flytekit complain if the `Resources` is not set here?

Reply:
yes, it throws an error. i've made the relevant change.